src/geom_handlers/common_assessment.jl

```diff
@@ -0,0 +1,235 @@
+""" Functions common to both site_assessment methods. """
+
+using LinearAlgebra
+
+include("geom_ops.jl")
+
+# Additional functions for reef-edge alignment processing.
```

Comment:
This comment seems to be loose? Should it be here or was it left here after a copy/paste by accident?